vonk/features/conformanceresources.rst

```diff
@@ -171,6 +174,8 @@ You are encouraged to manage and publish your profiles and related Conformance R
 You can load from multiple Simplifier projects by adding them to the list. The environment variable version of this is::
 
   VONK_Administration:SimplifierProjects:0:Uri=<FHIR endpoint for retrieving StructureDefinitions>
+  
+Vonk automatically find out about the FHIR version for each project and import it only for the matching FHIR version.
```

Comment:
Bad grammar here

Reply:
Suggestion:

"Vonk automatically finds the FHIR version for each project and imports it only for the matching FHIR version."